analysis/plotting.py

```diff
@@ -405,6 +405,56 @@ def write_heatmap_plot(self, p_values, image_path, symmetric=False):
                                   image_path,
                                   symmetric=symmetric)
 
+    @staticmethod
+    def _heatmap_plot(values, axes, args):
+        """Custom heatmap plot which mimics SciPy's sign_plot."""
+        args.update({'linewidths': 0.5, 'linecolor': '0.5', 'square': True})
+        # Annotate with values if less than 12 fuzzers.
```

Comment:
Can we deprecate the `heatmap_plot()` function above that calls `sp.sign_plot` in favor of this one? (Doesn't need to happen in the PR if yes)

Reply:
Yes, I will submit this change in a subsequent PR.